lib/FFI/Platypus/Lang/Win32.pm

```diff
@@ -193,15 +193,45 @@ sub native_type_map
       }
     }
 
-    # stuff we are not yet dealing with
-    # LPCTSTR is unicode string, not currently supported
-    # LPWSTR 16 bit unicode string
-    # TBYTE TCHAR UNICODE_STRING WCHAR
+    # LPWSTR is simply mapped to opaque. Here are some recipies:
```

Comment:
Can you explain why we treat `LPWSTR` differently from `LPCWSTR`?  Is it normally a convention that LPWSTR is not null terminated?

Reply:
The "C" that's missing is for "constant". LWPSTR is a modifiable buffer. You provide space, the function fills it in.